### PR TITLE
docs(api): document export method

### PR DESCRIPTION
**Pull request by SamVerschueren**
Documents the new `export()` method.

We have to wait until the latest version of `@webcontainer/api` is released.

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -988,6 +1042,42 @@ The size of the attached terminal.
...
+
+The format of the exported data. The `json` and `binary` format can be used as `tree` when calling [`mount`](#▸-mount).
+
+The default value is `json`.
+
+<br />
+
+#### ▸ `includes?: string[]`
+
+Globbing patterns to include files from within excluded folders.
```
Comment (Nemikolh) [inline docs/api.md:1071]: Oh gosh I had to read this twice to understand it :face_with_spiral_eyes: 

I don't think we can make it simpler though :smile:

**Review by Nemikolh — COMMENTED** [review]
Nice! Do you think you could also update the `changelog.md` file?

File: docs/api.md (hunk 1 above)
Comment (SamVerschueren) [reply docs/api.md:1071]: Yeah I know 😂 .

**Review by kirjavascript — APPROVED** [review]

**Comment by SamVerschueren** [comment]
Added the changelog.

**Review by Nemikolh — APPROVED** [review]
Yay! :star_struck:

File: docs/api.md [hunk 2]
```diff
@@ -310,6 +310,35 @@ Spawns a process with additional arguments.
 
 Spawns a process without additional arguments.
 
+### ▸ `export`
+
+Exports the filesystem.
```
Comment (Nemikolh) [inline docs/api.md:315]: Oh do you think you could add a "Added in 1.4.0" here?

**Review by Nemikolh — APPROVED** [review]

**Pull request merged**